app/components/primer/octicon_component.rb

```diff
@@ -33,6 +33,13 @@ class OcticonComponent < Primer::Component
     # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
     def initialize(icon_name = nil, icon: nil, size: SIZE_DEFAULT, use_symbol: false, **system_arguments)
       icon_key = icon_name || icon
+
+      # Don't allow sizes under 16px
+      if system_arguments[:height].present? && system_arguments[:height] < 16 || system_arguments[:width].present? && system_arguments[:width] < 16
```

Comment:
we should call this out in the docs

Reply:
also, extracting `16` to a constant like `MIN_SIZE` could be helpful for the future :)

Reply:
It's currently a secret that the component accepts height and width. but it shouldn't. Unfortunately this will require a lot of refactoring on dotcom to adjust to the proper sizes. This was an attempt to start cutting down what's possible. 

It would be awesome to make this more of an allowlist type thing where we list the sizes currently being used in dotcom and start chipping away at what's possible for custom sizes.